web_src/js/index.js

```diff
@@ -1201,6 +1201,12 @@ async function initRepository() {
   }
 }
 
+function initPullRequestMergeInstruction() {
+  $('.show-instruction').on('click', () => {
+    $('.instruct').toggle();
+  });
+}
```

Comment:
Could probably also do this without any JS by using the `<details>` and `<summary>` elements.

Reply:
Yes, it is. But the arrow is not that suitable here.

Reply:
You mean the triangle? That can be hidden:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#Customizing_the_disclosure_widget

Reply:
This is difficult to me and I will left this for other PRs.

Reply:
Yeah I guess it may be hard to have it only trigger from the link element.